docs/context.dox

```diff
@@ -255,7 +255,7 @@ of OpenGL ES extensions is identical except for the name of the extension header
 
 The `glext.h` extension header is a continually updated file that defines the
 interfaces for all OpenGL extensions.  The latest version of this can always be
-found at the [OpenGL Registry](http://www.opengl.org/registry/).  There are also
+found at the [OpenGL Registry](https://www.opengl.org/registry/).  There are also
```

Comment:
This page redirects to https://www.khronos.org/registry/OpenGL/index_gl.php but it still may be the correct one, it’s probably better to keep your version.